ui/lib/core/addon/components/confirm-action.js

```diff
@@ -14,50 +14,77 @@ import hbs from 'htmlbars-inline-precompile';
  *  </ConfirmAction>
  * ```
  *
- * @property {Func} onConfirmAction=null - The action to take upon confirming.
- * @property {String} [confirmMessage=Are you sure you want to do this?] - The message to display upon confirming.
+ * @property {Func} [onConfirmAction=null] - The action to take upon confirming.
+ * @property {String} [confirmTitle=Delete this?] - The title to display when confirming.
+ * @property {String} [confirmMessage=Are you sure you want to do this?] - The message to display when confirming.
  * @property {String} [confirmButtonText=Delete] - The confirm button text.
  * @property {String} [cancelButtonText=Cancel] - The cancel button text.
+ * @property {String} [disabledTitle=Can't delete this yet] - The title to display when the button is disabled.
  * @property {String} [disabledMessage=Complete the form to complete this action] - The message to display when the button is disabled.
  *
  */
 
 export default Component.extend({
-  tagName: 'span',
```

Comment:
What do you think about doing `tagName: ''` and then moving all class and wrapping HTML to the template?

Reply:
I'd have to pass in the data-attributes explicitly like I did for ToolbarLink (until we can use `...attributes`), right? Is that worth it?

Reply:
You should be able to use `...attributes` with this one because it's not one of the built-in template helpers (link-to was the problem with the other one iirc). But up to you if you want to change it - when we move to glimmer components that will be the default way things work so more just trying to shift to that where we can.